src/lib/features/context/context-service.ts

```diff
@@ -140,6 +140,45 @@ class ContextService {
         });
     }
 
+    async updateContextFieldLegalValue(
+        contextFieldLegalValue: { name: string; legalValue: LegalValueSchema },
+        auditUser: IAuditUser,
+    ): Promise<void> {
+        const contextField = await this.contextFieldStore.get(
+            contextFieldLegalValue.name,
+        );
+        const validatedLegalValue = await legalValueSchema.validateAsync(
+            contextFieldLegalValue.legalValue,
+        );
```

Comment:
Maybe I'm missing something, but why are we using the schema to validate here? Isn't that automatically taken care of at the endpoint level?

Reply:
Oh, also, `legalValueSchema` is not `LegalValueSchema` from `../../openapi`.  I *thought* it looked like Joi, but was a little confused by the import (and didn't look hard enough, apparently).

But everything that the joi schema does, we should be able to do with openapi directly, right? Set string min and max lengths and allow description to be `null`, `undefined`, or an arbitrary string?

Reply:
It's consistent with our current approach in the context module where openapi validates simple string and joi actual 1-100 characters. I didn't want to introduce a new convention in this PR

Reply:
Yeah, that's fair. I suspect we'll just forget about it and never touch it, then. But that's fine too. It's a little added complexity, but not a big deal here.